storage/scylladb_importer.c

```diff
@@ -148,6 +151,7 @@ int main(int argc, char* argv[]) {
   pthread_create(&importer_thread, NULL, (void*)importer_handler, (void*)&importer_data);
 
   pthread_join(importer_thread, NULL);
+  sleep(10);
```

Comment:
Can you avoid hardcoded `10` seconds?

Reply:
I add `db_permanode_tpool_wait` to wait until these are no tasks left in the pool and all workers do not perform tasks.